share/spack/setup-env.csh

```diff
@@ -41,7 +41,9 @@ if ($?SPACK_ROOT) then
 
     # Set up modules and dotkit search paths in the user environment
     # TODO: fix SYS_TYPE to something non-LLNL-specific
-    _spack_pathadd DK_NODE    "$_spack_share_dir/dotkit/$SYS_TYPE"
-    _spack_pathadd MODULEPATH "$_spack_share_dir/modules/$SYS_TYPE"
+    if ($?SYS_TYPE) then
+        _spack_pathadd DK_NODE    "$_spack_share_dir/dotkit/$SYS_TYPE"
+        _spack_pathadd MODULEPATH "$_spack_share_dir/modules/$SYS_TYPE"
+    endif
```

Comment:
You're right it isn't related to fastmath. But, is the additional overhead of sep. PR for two lines of code really worth it? I didn't think so. That aside, as I am relatively new to git, how would one achieve this? I assume I need to somehow remove this file change from this PR and then cherry pick just this file change to create a new PR. Is that right?